std_msgs/Header header

# layout: [x, y, z]
float32[3] index_points
float32[3] thumb_points
